tests/filestore-v2.10-wrong-direction/test.rules

```diff
@@ -0,0 +1,2 @@
+alert http any any -> any any (msg:"alert png images"; http.uri; content:".png"; nocase; sid:1; rev:1;)
+alert http any any -> any any (msg:"store png images"; http.uri; content:".png"; nocase; filestore; sid:2; rev:1;)
```

Comment:
I do not think this should match, the others look ok

Reply:
see discussion at https://github.com/OISF/suricata/pull/12312#issuecomment-2610083420